content/docs/for-platform-operators/how-to/build-inputs/build-env.md

```diff
@@ -0,0 +1,18 @@
++++
+title="Specify the build time environment variables"
+weight=4
++++
+
+Environment variables are any values used to allow `Buildpacks` configurability. Some environment variables can't be modified while others are expected to get changed to allow a level of customization.
+
+<!--more-->
+
+At `build` time, platform operators usually control what is included in the `build` environment because `platform-defined` environment variables override any previous values such as `user-provided` and `buildpack-provided` variables.
```

Comment:
```suggestion
The platform operator specifies environment variable in a manner that is very similar to buildpacks (see XXX), but with a few differences. Namely:
* The env directory is `/cnb/build-config`
* When no suffix is provided, the modification behavior is `default`
```

For more information, consult the [Platform Specification](https://github.com/buildpacks/spec/blob/main/platform.md).

Reply:
updated